pandas/tests/test_categorical.py

```diff
@@ -3852,16 +3852,15 @@ def test_concat(self):
         res = pd.concat([df, df])
         tm.assert_frame_equal(exp, res)
 
-        # Concat should raise if the two categoricals do not have the same
-        # categories
```

Comment:
IMO the old test case should be modified that it still again tests the case that ("more") unequal cats fail the concat.